app/scripts/formentry/controllers/formentry.controller.js

```diff
@@ -860,6 +860,7 @@ jshint -W098, -W003, -W068, -W004, -W033, -W030, -W117, -W069, -W106
                     .saveUpdatePersonAttribute(attribute, function (data) {
                         if (data) {
                             $log.log('Updated attribute: ', data);
+                            reloadPatientAfterUpdate(attribute.person.uuid());
```

Comment:
@kwatuha why don't you the returned data to update the patient object in the root scope instead of making another call to the server? I would prefer to reduce the number of backend calls as much as possible.